docs/tutorial/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye-slim
+FROM debian:bookworm-slim
 
 ARG PGVERSION=14
 ARG CITUS=postgresql-14-citus-11.1
```

Comment:
Can we also update that to Postgres 17 and Citus 13? That would fix https://github.com/hapostgres/pg_auto_failover/pull/1029 I believe.

Reply:
Added in 392f825